pluginManager/src/PluginList.cpp

```diff
@@ -383,10 +395,14 @@ void PluginList::addSteps(Plugin* plugin, TiXmlElement* installElement, InstallO
 	{
 		// If it is a unicode tag, then only process the contents if it's a unicode N++
 		// or if it's an ansi tag, then only process the contents if it's an ansi N++
-		if ((g_isUnicode 
-			&& !_tcscmp(installStepElement->Value(), _T("unicode")) 
+		if ((g_isUnicode && g_isX64

```

Comment:
Yes, as you mentioned X64 implicit means also unicode as notepad++ dropped ansi support before X64 arises. So if the structure at the server side is clear, I could adapt the naming here and extend the comment.